packages/minecraftBedrock/schema/general/vanilla/itemGroup.json

```diff
@@ -9,95 +9,85 @@
 		{
 			"type": "string",
 			"enum": [
-				"itemGroup.name.axe",
-				"itemGroup.name.pickaxe",
-				"itemGroup.name.shovel",
-				"itemGroup.name.sword",
-				"itemGroup.name.tools",
-				"itemGroup.name.hoe",
-				"itemGroup.name.trident",
-				"itemGroup.name.throwables",
-				"itemGroup.name.shield",
-				"itemGroup.name.arrow",
-				"itemGroup.name.bows",
-				"itemGroup.name.totem",
-				"itemGroup.name.natureMisc",
-				"itemGroup.name.sign",
+				"nature",
+				"construction",
+				"items",
+				"equipment",
+				"none",
+				"commands",
```

Comment:
These categories are already in https://github.com/bridge-core/editor-packages/blob/main/packages/minecraftBedrock/schema/general/vanilla/creativeCategory.json

Is there somewhere where these categories are valid and not referenced in the schemas?

Reply:
Yes; `parent` of `minecraft:creative_category` item component, and `group` of `minecraft:creative_category` block component

Reply:
👍 I'll sort this out and merge your changes